Directory.Build.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+ <PropertyGroup>
+   <Version>4.0.0</Version>
+   <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <Company>AgentFramework</Company>
+    <Authors>Tomislav Markovski, Tobias Looker</Authors>
+    <PackageProjectUrl>https://github.com/streetcred-id/agent-framework</PackageProjectUrl>
```

Comment:
Do we need to change this?

Reply:
I assume Company needs to be Hyperledger. Not sure about authors, perhaps the same as Company? @ryjones Any thoughts on this?

Reply:
I would update the PackageProjectUrl but not worry about the Company right now

Reply:
Updated all old url references, license with stock version, and removed binary files from history. We can squash and merge this PR.